WebDevStudios/wd_s
# Hotfix/#509 gutenberg columns

Closes #509

### DESCRIPTION ###
Fixes some backend styling for cases when our blocks are used in columns.

### SCREENSHOTS ###
#### Wide
![block displayed at wide-width](https://d.pr/i/RXkJFS+)

#### Full
![block displayed at full-width](https://d.pr/i/kFwJwp+)

#### Background Color
![block displayed with a background color](https://d.pr/i/lzrxII+)

### OTHER ###

### STEPS TO VERIFY ###
Add blocks to a Gutenberg column and see how it fits! I tested this with our blocks and things seem to look good. Setting the block full-width takes the content all the way to the edges, so you'll see the collision with the borders as displayed in the second image. To get around this, add a background color as seen in the third image which automatically adds additional padding to push the content away from the edges.

The width really only affects the backend, as on the frontend the blocks always fill the width of the column. I did, however, add some adjustments to the `.alignfull` class when inside of `.wp-block-column` to ensure it doesn't break out of the container.

@superfein – if you don't mind taking a look at this, I'd love to make sure this fixes any issues you were running into for the issue your originally reported.

Thanks!
